robot/description/launch/caffeine_gazebo.launch

```diff
@@ -2,6 +2,7 @@
    <arg name="use_gui" default="false"/>
    <arg name="debug_mode" default="false"/>
    <arg name="world" default="walls"/>
+   <arg name="camera_ns" default="/zed/zed_node"/>
```

Comment:
Given that this is the only place we call the ZED stereo camera, I don't think we need to add a `camera_ns` arg in `caffeine_gazebo.launch`